src/app/organizations/registerOrganization/register-organization.component.html

```diff
@@ -46,6 +46,11 @@ <h1 mat-dialog-title>{{title}}</h1>
       <mat-error *ngIf="contactEmail.hasError('email')">Please enter a valid contact email address</mat-error>
       <mat-hint>Once approved, this email address will be publicly visible</mat-hint>
     </mat-form-field>
+    <mat-form-field>
+      <input type="text" matInput formControlName="avatarUrl" placeholder="Image URL">
+      <mat-error *ngIf="avatarUrl.hasError('pattern')">Link must end in jpg, jpeg, or png</mat-error>
+      <mat-hint>Link to your organization's logo image. Link must end in jpg jpeg or png</mat-hint>
```

Comment:
1. `logo image` seems redundant. I would just say `logo`.
2. Don't you now support gif (applies to previous line too)?
3. There should be commas like you have in the previous line.